site/src/js/index.js

```diff
@@ -1,103 +1,106 @@
-/* global Selectr */
+/* global window, document, Selectr */
 
 /**
  * Initialising global variables
  */
 const ACTIVE_CLASS = 'visible';
 const sort = {
-    ALPHABETICAL_ASCENDING: (a,b) => a.dataset.name > b.dataset.name ? 1 : -1,
-    ALPHABETICAL_DESCENDING: (a,b) => a.dataset.name < b.dataset.name ? 1 : -1,
-    DIFFICULTY_ASCENDING: (a,b) => a.dataset.difficulty > b.dataset.difficulty ? 1 : -1,
-    DIFFICULTY_DESCENDING: (a,b) => a.dataset.difficulty < b.dataset.difficulty ? 1 : -1
+  ALPHABETICAL_ASCENDING: (a, b) => a.dataset.name > b.dataset.name ? 1 : -1,
+  ALPHABETICAL_DESCENDING: (a, b) => a.dataset.name < b.dataset.name ? 1 : -1,
+  DIFFICULTY_ASCENDING: (a, b) => a.dataset.difficulty > b.dataset.difficulty ? 1 : -1,
+  DIFFICULTY_DESCENDING: (a, b) => a.dataset.difficulty < b.dataset.difficulty ? 1 : -1
 };
 
 const contentEl = document.getElementById('content');
 const filterInput = document.getElementById('filter');
 const sortingInput = document.getElementById('sorting');
 
-const activateElements = els => Array.from(els).forEach(node => node.classList.add(ACTIVE_CLASS));
+const activateElements = els => [...els].forEach(node => node.classList.add(ACTIVE_CLASS));
 const allowDifficultySelect = shouldAllow => sortingInput.querySelectorAll('.difficulty')
-    .forEach(node => node.disabled = !shouldAllow);
+  .forEach(node => {
```

Comment:
Was there a specific reason for multi-lining this? It felt legible on a single line 😉

Reply:
This is also default with xo: https://github.com/xojs/eslint-config-xo/blob/master/index.js#L116

Reply:
Aah, forgot about that 🤮 could you try adding braces around it? I don't remember if that would trigger a `one-statement-per-line`-like error

`.foreach(node => {node.disabled = !shouldAllow})`

Reply:
Yes it triggers it, that's why I multi-lined it :/

Reply:
Oh well, you win some you lose some! 😄